src/style.css

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.photo {
+  height: 40px;
+}
+
+nav ul {
+  list-style-type: none;
+  display: flex;
+  padding: 0;
+}
+
+nav li {
```

Comment:
Fix it everywhere, you need to use the classes selector
<img width="770" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/cce3ca28-bd85-4104-ba8a-b5509422cd35">